pom.xml

```diff
@@ -87,6 +89,17 @@
                 <artifactId>snakeyaml</artifactId>
                 <version>${snakeyaml.version}</version>
             </dependency>
+            <!-- band aid until the connector catches up with updated pom parent -->
+            <dependency>
```

Comment:
snappy-java dependency is resolved in this PR -
https://github.com/confluentinc/kafka-connect-storage-cloud/pull/669/files

Reply:
it was done in parallel. please note update to jetty.